### ISSUE TITLE
Make easier to separate the training and testing data

### ISSUE THREAD

**Issue by fhtanaka** [fhtanaka/directed_research_CS_2018#4]
There should be a variable to determine the size of each

**Comment by fhtanaka** [comment]
Maybe it is a good idea to shuffle the data training data between epochs
